Assets/MixedRealityToolkit/Inspectors/MixedRealityPreferences.cs

```diff
@@ -67,7 +67,7 @@ public static bool IgnoreSettingsPrompt
         [SettingsProvider]
         private static SettingsProvider Preferences()
         {
-            var provider = new SettingsProvider("Project/MRTK")
+            var provider = new SettingsProvider("Project/MRTK", SettingsScope.Project)
```

Comment:
The settings should stay in the preferences window, not be moved to the project settings

```suggestion
            var provider = new SettingsProvider("Project/MRTK", SettingsScope.User)
```

Reply:
This doesn't move anything - the description for this PR shows how the old code defaulted to SettingScope.Project. I pointed out the same issue on:

https://github.com/Microsoft/MixedRealityToolkit-Unity/issues/3951

Reply:
Yeah I accidentally moved it when I updated the preferences window a few weeks back.

Believe me, it used to be in preferences when I first wrote it, and these settings have nothing to do with the project build settings.

Reply:
Having things like profile locks at the project level feels right. Developers may wish to unlock profiles for only a specific project to avoid accidentally modifying them in others.

Reply:
Then it might be better to actually split up this window as there are both project and non project specific settings.